### PR TITLE
team_datasnokmennene

### PR DESCRIPTION
**Pull request by tweey**
team: datasnokmennene
public key: MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA36Z1FbnSzgctY3BUr2G1
xsXy9k3VlR1ivXVdsXn4WxB5os6uAM19xasI58IzOKNkEbxc1UzozlzAWNc0w+2b
vANTMzM8Kg/4iuTeUcLgfu6ICuUHlNWdt4gVubpVs9037COvv5+jpv9l+mIWchY5
Qm7siZqHpCTLs5TogjQSwkUvroCIDkVNG+bttTkK8OX4sQfAxv3wmmxNPExhy+yt
dlYZHPKxYA+ehsG/U/RkZnXYzda3gHv/uR/MktLXbTmcPsAP7kDO9bfqi4pQQsnl
r8oJz00H+bf+OtDY7igOtM/4fLHSZsIpF3a7YLPtg2odUT4S3zExjOxuTMRvJAPD
nwIDAQAB

### CODE REVIEW

File: team_datasnokmennene/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+```python
+import drawer
+import fridge
+import time
+import appliances	
+
+def mix_butter_and_sugar(butter, sugar):
+    mix = appliances.mix(butter, sugar)
+    mix.append(vanilla*0.05)
```
Comment (StevenVanAcker) [inline team_datasnokmennene/index.md]: what units are these?

File: team_datasnokmennene/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+def cut_chocolate(chocolate)
+    bits = appliances.knife(chocolate)
+    return bits
+
+def main():
+    oven.temp(160)
+    oven.mode(over_and_under_heat)
+
+    # Making the dough
+    dough = mix_butter_and_sugar(drawer.sugar*250, fridge.butter*250)
```
Comment (StevenVanAcker) [inline team_datasnokmennene/index.md]: grams?

File: team_datasnokmennene/index.md (hunk 1 above)
Comment (tweey) [reply team_datasnokmennene/index.md]: I fixed the units. Please review again :D

File: team_datasnokmennene/index.md (hunk 2 above)
Comment (tweey) [reply team_datasnokmennene/index.md]: Yes. I fixed the code and added units

**Comment by StevenVanAcker** [comment]
Thank you!!

BvAOeGvwb6kxdSCshrtlUTdgOk2uymcqjegEzJivphEmlKeoj5llDcDEUjhaQeQJTfGsqtEQtA4U
/q7er3JdK+OLFfMSQ2qxhE2G8JVI3nJY++7RbTj3CVjAt7cTBeLB+ZTIKi5i1bk3rrFBvGCEipmA
zmQeswYfjFfOhmUG/Ii9EFkZL/1IBjVRZqfZMAJk5lxJDWiiAKJ6ZGciR56Sd3zxIP645Qide1PS
8Fu6J8ycUYdfp3qg/hjoP+0GlNFFyArxHw2p3r/FE85YzZXWxCRy2CNd7YFTCFGSRcg/w6XnXjys
NdUSQyNwyM7DdiCuOYMVaPXNwd7jJjeQb7Fe5Q==

**Comment by StevenVanAcker** [comment]
0cqT9k1p+jFHM9l2pt8Scj6uSQ2qDiw1ji2O9g4jH9D7TmZsxnqqDfEN5QZhJWOaHatc9HWCLAn0
YfZ8Ticv0BwPmOdVSak2SpRMY0Yi6+g6zEk5R939UVVJAZHgHktrfqARZzpgi8uT9YiepLsygMJs
moFnWz/DNI5iLkxO82A1e8Ri+Wi8lN0BtKoC0nRq9sm4Je5FNIZA1xdhQp3bd+OCzx3C2QfebAlA
cwhirXmpYHpXhe8zIdTXwDokNQTbNpnrxx8WF85Ot554g91aMIP+aybtZkMEGRqAsqNGF2N6muBt
aKjYRE3k6aXA3kzAF/wLUhGG3oGPxrEK8NQm+A==

**Pull request merged**